en/lessons/basics/collections.md

```diff
@@ -3,120 +3,224 @@ version: 1.3.1
 title: Collections
 ---
 
-Lists, tuples, keyword lists, and maps.
+Tuples, Lists, Assoc Lists, Sets, Maps/Hashmaps, Seqs, and Vectors.
 
 {% include toc.html %}
 
+## Tuples
+
+Tuples are the first kind of collection you are introduced to in haskell. This is because they are simple, primitive, and have a terse, flexible, built in syntax. Tuples are sometimes referred to as anonymous records, but instead of referencing fields by name you reference them by position in the structure. Tuples can contain an arbitrary number of items, but we will focus on the 2-tuple since haskell has good built in support for them. However, if you are interested this is what an 8-tuple looks like
+
+```haskell
+ghci> :t ('0','1','2','3','4','5','6',"8-tuple")
+('0','1','2','3','4','5','6',"8-tuple")
+  :: (Char, Char, Char, Char, Char, Char, Char, [Char])
```

Comment:
Type declarations have spaces after commas, but terms do not. It would be nice to make it consistent (I suggest putting spaces in terms as well).